mithril.js

```diff
@@ -28,6 +28,19 @@
 		return "v0.2.5"
 	}
 
+	var div = document.createElement("div")
+	var frag = document.createDocumentFragment()
+
+	function toDOM(HTML){
+		div.innerHTML = HTML
+
+		for (var i = 0; i < div.childNodes; i++) {
```

Comment:
`for (var i = 0; i < div.childNodes; i++) {` vs `for (var i = 0; i < div.childNodes.length; i++) {` could be another source of issues? (`.length`)